config/manager/manager.yaml

```diff
@@ -31,8 +31,8 @@ spec:
         resources:
           limits:
             cpu: 100m
-            memory: 90Mi
+            memory: 180Mi
           requests:
             cpu: 100m
-            memory: 60Mi
+            memory: 90Mi
```

Comment:
Should we document the rational behind these numbers, and how/if/when to adjust them?

Reply:
The rationale is that the operator was crashing because it was hitting the memory limit. Now, it's not crashing. So if the operator is crashing due to hitting the memory limit, it a bug on our part and a ticket should be opened to fix this because it means a newly fixed build and release.

Reply:
How did we establish 90Mi is the right initial memory? Under what situations would customers want to tune this up or down? Are there docs we can reference to help customers in this decision?

As an example, if a container has extremely high activity it will generate far more detections/events than a idle web server. At what point should these values be increased? Why are they not the same as the recommended minimum memory for a Linux VM?

Reply:
> How did we establish 90Mi is the right initial memory? Under what situations would customers want to tune this up or down? Are there docs we can reference to help customers in this decision?

We established it in our test cluster. Kubernetes will scale the resources up and down to its limit based on what the controller manager needs. Since this is for the operator manager pod, this really shouldn't be configured. If anything, I could probably lower the initial requests and let the cluster scale up/down the memory, and I should probably set a limit for cpu requests.

> As an example, if a container has extremely high activity it will generate far more detections/events than a idle web server. At what point should these values be increased? Why are they not the same as the recommended minimum memory for a Linux VM?

This is for the operator manager pod and not a sensor pod